source/api_reference/version-history.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Version history
+weight: 0
+---
+# Version history
+
+## Version 1
+
+Version 1 can be accessed using either of these base URLs:
+
+- `https://{register-name}.register.gov.uk/v1/`
+- `https://{register-name}.register.gov.uk/`
+
+## Next version
+You can use this version of the API to test against backwards incompatable API changes before we release a new version of the API.
+
+Access it using this base URL:
+
+- `https://{register-name}.register.gov.uk/next/`
+
+**WARNING: Do not use this URL in production, as its API may change without warning.**
+
+### Changes since version 1
```

Comment:
I'm not sure about "Changes since Version 1" if 'Next' API is to be a rolling term. It also may end up being hard to maintain this and this section may slip into obsolescence if nobody tends to it regularly.

Reply:
I was intending this to work like the "unreleased" sections in our changelogs, so that we can document the next version straight away, and then we just rename the heading when `/next` becomes `/v2` or `/v3`.

I'll have a think about how to make this easier to maintain. There are various other things we need to remember to do when making breaking changes as well as this, so it would be great to have a checklist or something.

Reply:
There's nothing wrong with it in principle; it's just in my experience of working on different projects, "Version history" sections sometimes end up being very outdated.